_includes/components/universal/LineChart/index.jsx

```diff
@@ -20,7 +20,13 @@ export default function LineChart(props) {
 
 
   if (!width) {
-    return <div className="LineChart" ref={node => getNode && getNode(node)} />;
+    const loader = <div className="Loader u-marginTop50 u-marginLeftAuto u-marginRightAuto u-marginBottom40" />;
+
+    return (
+      <div className="LineChart" ref={node => getNode && getNode(node)}>
+        {loader}
```

Comment:
In the past I've used `::empty` and `::before` pseudo element to add loading indicators without JS.

Reply:
Yeah, I'm using the same principle, but just attaching the styling to a HTML node itself instead of the `::before` property of the node. The animation itself is still done without JS.